tests/CMakeLists.txt

```diff
@@ -122,7 +121,7 @@ add_test_case(h1_client_new_request_allowed)
 add_test_case(h1_client_midchannel_sanity_check)
 add_test_case(h1_client_midchannel_read)
 add_test_case(h1_client_midchannel_read_immediately)
-add_test_case(h1_client_midchannel_read_with_small_downstream_window)
+#add_test_case(h1_client_midchannel_read_with_small_downstream_window)
```

Comment:
Why disable this test?

Reply:
whoa, good catch.
Turned it back on, it works.
I'd disabled all "window" tests at some point and forget to turn this back on